pkgs/applications/office/nanonote/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, qtbase
+, qttools
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nanonote";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    owner = "agateau";
+    repo = "nanonote";
+    rev = "${finalAttrs.version}";
+    hash = "sha256-MsVHu3lAe/aGzFt1xDrsZHzLF1ysjhRUfruypoXEEnU=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    extra-cmake-modules
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    qtbase
+    qttools
+
+  ];
+
+  buildPhase = ''
+    cmake ..
+  '';
+
+  installPhase = ''
+    make install PREFIX=$out
+  '';
```

Comment:
Please add preBuild/postBuild and preInstall/postInstall hooks using runhook here. take `pkgs/applications/misc/tandoor-recipes/frontend.nix` as a example